src/constants/routes.ts

```diff
@@ -208,3 +208,11 @@ export const getCommitQueueRoute = (projectId: string) =>
 
 export const getCommitsRoute = (projectId: string) =>
   `${paths.commits}/${projectId}`;
+
+export const getVariantHistoryRoute = (
+  projectId: string,
```

Comment:
I think we should actually label this projectIdentifier since that's what we're actually passing through; in the future we may otherwise rely on this being ID when it's actually the mutable identifier.